specification_document-developments/2_0-Metabolomics-Draft/mzTab_2_0-M_mapping.xml

```diff
@@ -0,0 +1,171 @@
+        </CvMappingRule>
+        
+        <CvMappingRule id="instrument_detector_may" cvElementPath="/metadata/instrument/@instrumentDetector" requirementLevel="MAY"  scopePath="/metadata/instrument" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="MS:1000026" useTerm="false" termName="detector type" isRepeatable="false" allowChildren="true" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule>
+        
+        <!-- software -->
+        <CvMappingRule id="software_must" cvElementPath="/metadata/software/@parameter" requirementLevel="MUST"  scopePath="/metadata/software" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="MS:1002878" useTerm="false" termName="small molecule analysis software" isRepeatable="false" allowChildren="true" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule>
+        
+        <!-- quantification method -->
+        <CvMappingRule id="quantification_method_must" cvElementPath="/metadata/@quantificationMethod" requirementLevel="MUST"  scopePath="/metadata" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="PRIDE:0000307" useTerm="false" termName="Quantification method" isRepeatable="false" allowChildren="true" cvIdentifierRef="PRIDE"></CvTerm>
+        </CvMappingRule>
+        
+        <!-- sample -->
+        <CvMappingRule id="sample_species_may" cvElementPath="/metadata/sample/@species" requirementLevel="MAY"  scopePath="/metadata/sample" cvTermsCombinationLogic="OR">
+          <CvTerm termAccession="PRIDE:0000033" useTerm="false" termName="NEWT" isRepeatable="false" allowChildren="true" cvIdentifierRef="PRIDE"></CvTerm>
+          <CvTerm termAccession="NCBITaxon:1" useTerm="false" termName="root" isRepeatable="false" allowChildren="true" cvIdentifierRef="NCBITaxon"></CvTerm>
+        </CvMappingRule>
+        
+        <CvMappingRule id="sample_tissue_may" cvElementPath="/metadata/sample/@tissue" requirementLevel="MAY"  scopePath="/metadata/sample" cvTermsCombinationLogic="OR">
+          <CvTerm termAccession="BTO:0000000" useTerm="false" termName="tissues, cell types and enzyme sources" isRepeatable="false" allowChildren="true" cvIdentifierRef="BTO"></CvTerm>
+          <CvTerm termAccession="PRIDE:0000442" useTerm="false" termName="Tissue not applicable to dataset" isRepeatable="false" allowChildren="true" cvIdentifierRef="PRIDE"></CvTerm>
+        </CvMappingRule>
+        
+        <CvMappingRule id="sample_cell_type_may" cvElementPath="/metadata/sample/@cellType" requirementLevel="MAY"  scopePath="/metadata/sample" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="CL:0000000" useTerm="false" termName="cell" isRepeatable="false" allowChildren="true" cvIdentifierRef="CL"></CvTerm>
+        </CvMappingRule>
+        
+        <CvMappingRule id="sample_disease_may" cvElementPath="/metadata/sample/@disease" requirementLevel="MAY"  scopePath="/metadata/sample" cvTermsCombinationLogic="OR">
```

Comment:
"Disease" is fairly specific, and yet expressivity rather low (How severe the disease ? Post-Mortem ??) etc. I'd think that mzTab would not be the best place, but might be a good hint about a study.
Still, we can leave that here.